# HG changeset patch
# User Language Forge
# Date 1490983555 -7200
#      Fri Mar 31 20:05:55 2017 +0200
# Branch 7000068
# Node ID 7e948bdeaa3be62f599fa781c77382a271506fd6
# Parent  5436f177f0d75fa172135d0949a86749a4b73875
[LfMergeBridge: 2.5.1.0] Language Forge: 1 entry added, 1 entry deleted

diff -r 5436f177f0d7 -r 7e948bdeaa3b Linguistics/Lexicon/Lexicon_02.lexdb
--- a/Linguistics/Lexicon/Lexicon_02.lexdb	Fri Mar 31 20:04:43 2017 +0200
+++ b/Linguistics/Lexicon/Lexicon_02.lexdb	Fri Mar 31 20:05:55 2017 +0200
@@ -1,55 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon>
-	<LexEntry
-		guid="6e24c0b7-6b89-4097-9ddf-79a9e7281e4b">
-		<DateCreated
-			val="2017-3-21 17:8:9.236" />
-		<DateModified
-			val="2017-3-31 18:4:28.130" />
-		<DoNotUseForParsing
-			val="False" />
-		<HomographNumber
-			val="0" />
-		<LexemeForm>
-			<MoStemAllomorph
-				guid="a6eafc28-d389-48c8-a46b-197c9839ad89">
-				<Form>
-					<AUni
-						ws="fr">flexmodified</AUni>
-				</Form>
-				<IsAbstract
-					val="False" />
-				<MorphType>
-					<objsur
-						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
-						t="r" />
-				</MorphType>
-			</MoStemAllomorph>
-		</LexemeForm>
-		<MorphoSyntaxAnalyses>
-			<MoStemMsa
-				guid="a8ab8713-6db3-443b-8a9e-d956bd2c4d9c">
-				<PartOfSpeech>
-					<objsur
-						guid="46e4fe08-ffa0-4c8b-bf98-2c56f38904d9"
-						t="r" />
-				</PartOfSpeech>
-			</MoStemMsa>
-		</MorphoSyntaxAnalyses>
-		<Senses>
-			<ownseq
-				class="LexSense"
-				guid="6db6406c-bec9-4173-a464-29fbb5ace2e9">
-				<Gloss>
-					<AUni
-						ws="en">created in FLEx</AUni>
-				</Gloss>
-				<MorphoSyntaxAnalysis>
-					<objsur
-						guid="a8ab8713-6db3-443b-8a9e-d956bd2c4d9c"
-						t="r" />
-				</MorphoSyntaxAnalysis>
-			</ownseq>
-		</Senses>
-	</LexEntry>
-</Lexicon>
\ No newline at end of file
+<Lexicon />
\ No newline at end of file
diff -r 5436f177f0d7 -r 7e948bdeaa3b Linguistics/Lexicon/Lexicon_06.lexdb
--- a/Linguistics/Lexicon/Lexicon_06.lexdb	Fri Mar 31 20:04:43 2017 +0200
+++ b/Linguistics/Lexicon/Lexicon_06.lexdb	Fri Mar 31 20:05:55 2017 +0200
@@ -1,2 +1,102 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon />
\ No newline at end of file
+<Lexicon>
+	<LexEntry
+		guid="51fda335-51f1-46f5-abd1-9f9ce5e8f4df">
+		<DateCreated
+			val="2017-2-23 16:40:52.81" />
+		<DateModified
+			val="2017-3-31 18:5:18.457" />
+		<DoNotUseForParsing
+			val="False" />
+		<Etymology>
+			<LexEtymology
+				guid="c446a56b-77c1-42e1-ae54-9118bb429533" />
+		</Etymology>
+		<HomographNumber
+			val="0" />
+		<LexemeForm>
+			<MoStemAllomorph
+				guid="49a7c8fd-ebf7-4e4c-9486-0acb05f6bfe5">
+				<Form>
+					<AUni
+						ws="fr">lf1modified</AUni>
+				</Form>
+				<IsAbstract
+					val="False" />
+				<MorphType>
+					<objsur
+						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
+						t="r" />
+				</MorphType>
+			</MoStemAllomorph>
+		</LexemeForm>
+		<MorphoSyntaxAnalyses>
+			<MoStemMsa
+				guid="a32279f0-d700-43b5-ac95-266e1ff7caa5">
+				<PartOfSpeech>
+					<objsur
+						guid="a8e41fd3-e343-4c7c-aa05-01ea3dd5cfb5"
+						t="r" />
+				</PartOfSpeech>
+			</MoStemMsa>
+		</MorphoSyntaxAnalyses>
+		<Pronunciations>
+			<ownseq
+				class="LexPronunciation"
+				guid="5388cb7d-caa7-412f-9d73-a5315874ff2a" />
+		</Pronunciations>
+		<Senses>
+			<ownseq
+				class="LexSense"
+				guid="fcab1080-000b-4393-b95e-2631310d317d">
+				<Definition>
+					<AStr
+						ws="en">
+						<Run
+							ws="en">Word added by LF</Run>
+					</AStr>
+				</Definition>
+				<Examples>
+					<ownseq
+						class="LexExampleSentence"
+						guid="02544948-0bba-4034-8978-a72464955177">
+						<Example>
+							<AStr
+								ws="fr">
+								<Run
+									ws="fr">C'est un exemple</Run>
+							</AStr>
+						</Example>
+						<Translations>
+							<CmTranslation
+								guid="7f6242d6-72b8-4c70-a32b-a2d9295bd26b">
+								<Translation>
+									<AStr
+										ws="en">
+										<Run
+											ws="en">This is an example</Run>
+									</AStr>
+								</Translation>
+								<Type>
+									<objsur
+										guid="d7f7164a-e8cf-11d3-9764-00c04f186933"
+										t="r" />
+								</Type>
+							</CmTranslation>
+						</Translations>
+					</ownseq>
+				</Examples>
+				<MorphoSyntaxAnalysis>
+					<objsur
+						guid="a32279f0-d700-43b5-ac95-266e1ff7caa5"
+						t="r" />
+				</MorphoSyntaxAnalysis>
+				<SemanticDomains>
+					<refcol
+						guid="999581c4-1611-4acb-ae1b-5e6c1dfe6f0c"
+						t="r" />
+				</SemanticDomains>
+			</ownseq>
+		</Senses>
+	</LexEntry>
+</Lexicon>
\ No newline at end of file
diff -r 5436f177f0d7 -r 7e948bdeaa3b WritingSystemStore/en-US.ldml
--- a/WritingSystemStore/en-US.ldml	Fri Mar 31 20:04:43 2017 +0200
+++ b/WritingSystemStore/en-US.ldml	Fri Mar 31 20:05:55 2017 +0200
@@ -4,7 +4,7 @@
 		<version
 			number="" />
 		<generation
-			date="2017-03-22T15:17:27" />
+			date="2017-03-30T15:39:16" />
 		<language
 			type="en" />
 		<territory
diff -r 5436f177f0d7 -r 7e948bdeaa3b WritingSystemStore/en.ldml
--- a/WritingSystemStore/en.ldml	Fri Mar 31 20:04:43 2017 +0200
+++ b/WritingSystemStore/en.ldml	Fri Mar 31 20:05:55 2017 +0200
@@ -4,7 +4,7 @@
 		<version
 			number="" />
 		<generation
-			date="2017-03-21T16:45:47" />
+			date="2017-03-23T17:08:38" />
 		<language
 			type="en" />
 	</identity>
diff -r 5436f177f0d7 -r 7e948bdeaa3b WritingSystemStore/fr.ldml
--- a/WritingSystemStore/fr.ldml	Fri Mar 31 20:04:43 2017 +0200
+++ b/WritingSystemStore/fr.ldml	Fri Mar 31 20:05:55 2017 +0200
@@ -4,7 +4,7 @@
 		<version
 			number="" />
 		<generation
-			date="2017-03-21T16:45:47" />
+			date="2017-03-23T17:08:38" />
 		<language
 			type="fr" />
 	</identity>
